example/e2e/detoxTest.spec.js

```diff
@@ -185,7 +185,7 @@ describe('Example', () => {
       await userChangesMinuteValue();
       await userTapsOkButtonAndroid();
 
-      await expect(timeText).toHaveText('22:30');
+      await expect(timeText).toHaveText('23:30');
```

Comment:
I'd love to have a single `await expect(timeText).toHaveText(...);` so that we know this works the same on both platforms, would you please modify the e2e test to work this way? Thank you!